### ISSUE TITLE
Issue with custom progress columns not displaying properly when running from Docker

### ISSUE THREAD

**Issue by vadysa** [spectreconsole/spectre.console#1292]
**Information**
 - OS: [Windows/Linux]
 - Version: [0.47.0]
 - Terminal: [Docker Terminal]

**Describe the bug**
When running the C# code using AnsiConsole.Progress() with custom progress columns on both Windows and Linux, it works as expected. However, when running the same code within a Docker container, the custom progress columns do not appear, and it seems to use default settings.

**To Reproduce**

1) Run the C# code using AnsiConsole.Progress() with custom progress columns on a Windows or Linux host machine. Observe that the custom columns are displayed correctly.
2) Run the same C# code within a Docker container, and observe that the custom columns are not displayed as expected.

**Expected behavior**
When running the C# code within a Docker container, the custom progress columns should be displayed just like on a Windows or Linux host machine.

**Screenshots**
Windows:    
![image](https://github.com/spectreconsole/spectre.console/assets/53491873/bdef717f-acf5-40c3-af43-f424155695f7)     
Docker:   
![image](https://github.com/spectreconsole/spectre.console/assets/53491873/80718223-406e-4eb2-b7ae-d27cdec3d3f3)

**Additional context**
It appears that there might be an issue or misconfiguration specific to running the code within a Docker container that causes the custom progress columns to not be displayed. Investigating Docker-related configurations and environment settings may be necessary to resolve this issue.

 ```js
 AnsiConsole.Progress()
             .Columns(new ProgressColumn[]
             {
                new TaskDescriptionColumn(),
                new ProgressBarColumn(),
                new PercentageColumn(),
                new RemainingTimeColumn(),
                new SpinnerColumn(),
                new TransferSpeedColumn(),
             })
             .Start(ctx =>
             {
  ```

---
Please upvote :+1: this issue if you are interested in it.

**Comment by patriksvensson** [comment]
We could however consider making the columns configurable for the fallback renderer as well.

**Comment by isaacrlevin** [comment]
Jumping on this to see is there are workarounds. Would clearing the previous line be an option? In my case I do want to see progress, but I really don't need the thins like the spinner or bar, I'm fine with a percentage.

**Comment by FrankRay78** [comment]
Do you have a code snippet you can share @isaacrlevin showing how you've configured `AnsiConsole.Progress()` ?

**Comment by patriksvensson** [comment]
@mitchdenny Absolutely. I agree that the fallback behaviour should not show % for an indeterminate progress. A pull request would be very appreciated.

**Comment by Marches0** [comment]
Is this still under consideration? Having a line for each entry as opposed to a single one which updates is understandable in this scenario, but it'd be nice if the existing progress columns (`RemainingTimeColumn` etc.) could still be rendered, since they still provide useful information even if spread across multiple lines.
